CMakeLists.txt

```
@@ -30,6 +30,12 @@ if(QEMU)
   add_definitions(-D__QEMU__=1)
 endif()
 
+option(ASAN "Enable code asan" OFF)
```

```suggestion
option(ASAN "Enable AddressSanitizer (ASan)" OFF)
```
Probably a little clearer.